test/helpers.js

```diff
@@ -0,0 +1,76 @@
+import test from 'ava';
```

Comment:
I believe it's not a big enough project and not intended to ever be for this kind of thorough testing

> If main test failed, but unit - not --- first should check code not covered with tests.

It would make a sense if unit tests are separated and are ran firsts (no need to do it now)